PatchKit/Classes/Views/Cells/MessageTableViewCell.swift

```diff
@@ -0,0 +1,202 @@
+//
+//  MessageTableViewCell.swift
+//  Pods
+//
+//  Created by Omar Rasheed on 5/24/20.
+//
+
+import UIKit
+
+class MessageTableViewCell: UITableViewCell {
+
+    static let reuseIdentifier = "MessageTableViewCell"
+    
+    private let messageBubble = UIView()
+    private let messageLabel = UILabel()
+    private let adminImageView = UIImageView()
+    private let timestampLabel = UILabel()
+    private let imagesStackView = ImagesStackView()
+    private let readLabel = UILabel()
+    
+    private let verticalPadding: CGFloat = 6
+
+    override init(style: UITableViewCell.CellStyle, reuseIdentifier: String?) {
+        super.init(style: style, reuseIdentifier: reuseIdentifier)
+        
+        setupAdminImageView()
+        setupMessageBubble()
+        setupMessageLabel()
+        setupTimestampLabel()
+        setupImagesStackView()
+        setupReadLabel()
+    }
+    
+    func setupReadLabel() {
```

Comment:
You made all the functions in the `CustomerServiceDetailViewController` private. Any reasons why that's not the case here?

Reply:
Forgot to do that before opening PR. Will fix. An aside tho, we’re gonna wanna do this for the rest of the functions wherever possible